lib/util/package.py

```diff
@@ -0,0 +1,50 @@
+import os
+from pathlib import Path
+
+from .. import versions
+from .subprocess import subprocess_run
+
+
+def download_source(package):
+    """
+    Download src rpm.
+
+    'package' is name of package to download.
+    """
+    cmd = ['yumdownloader', '--source', package]
+    subprocess_run(cmd, check=True)
+    package_glob = package + "*.src.rpm"
+    try:
+        src_path = next(Path('.').glob(package_glob))
+    except StopIteration:
+        return None
```

Comment:
Actually, I would throw a `FileNotFoundError` here. The test code doesn't check for `None` and we probably want to abort if this operation fails anyway.